torch/_dynamo/eval_frame.py

```diff
@@ -921,7 +921,9 @@ def result_capturing_wrapper(*graph_inputs):
             constraint_violation_error = e
     remove_from_cache(f)
 
-    if (shape_env := getattr(fake_mode, "shape_env", None)) is not None:
+    if (
+        shape_env := getattr(fake_mode, "shape_env", None)
+    ) is not None and not skipfiles.check(inspect.getsourcefile(f)):
```

Comment:
What will happen if we don't do the checking? It'd great if we add some comment here.

Reply:
When f is skipped we do `wrap_inline` on it which turns the signature into `*args, **kwargs`. But for skipped files it's not worth suggesting constraints anyway. I'll add a comment.